kas/patches/meta-imx_meta-bsp_layerdir.patch

```diff
@@ -0,0 +1,10 @@
+diff --git a/meta-bsp/conf/layer.conf b/meta-bsp/conf/layer.conf
+index a161ce65fd..771893d0d5 100644
+--- a/meta-bsp/conf/layer.conf
++++ b/meta-bsp/conf/layer.conf
+@@ -425,3 +425,5 @@ IMXBOOT_TARGETS_SD:prepend:imx8dxl-ddr3l-evk = \
+ 
+ # Create a mirror for unreliable packages
+ MIRRORS += "http://sourceforge.net/.* http://www.nxp.com/lgfiles/updates/other"
++
++LAYERDIR_fsl-bsp-release = "${LAYERDIR}"
```

Comment:
out of curiosity: is this approach of referencing foreign meta layers common practice or is it our invention?
at least it doesn't seem to be an official or even documented way, my search in meta layers used by us did only show up the locations we patch ourselves in.

Reply:
our approach. idea was to find files independent of their repository location (which changed in the past and we had to adapt in code then).
do you have a proposal how to handle it differently?

Reply:
no, I don't, as stated I was only curious whether there was a blueprint for that somewhere or we (most likely you, actually ;-)) came up with that solution.